CriteoPublisherSdk/Sources/Feedback/CR_FeedbackStorage+MessageUpdating.h

```diff
@@ -27,7 +27,7 @@
                          profileId:(NSNumber *)profileId
                     requestGroupId:(NSString *)requestGroupId;
 
-- (void)setCdbEndAndCacheBidUsedIdForImpressionId:(NSString *)impressionId;
+- (void)setCdbEndAndCacheBidUsedIdForImpressionId:(NSString *)impressionId zoneId:(NSNumber *)id1;
```

Comment:
Should you format the signature similarly to `setCdbStartForImpressionId `?

Reply:
You mean with newlines ?

The formatting is done (and checked) automatically via a clang formatter. There is no many choice in this.

Reply:
See https://github.com/criteo/ios-publisher-sdk/actions/runs/196128466/workflow#L59 for the check.
And https://github.com/criteo/ios-publisher-sdk/blob/EE-1125/fastlane/Fastfile#L41 (triggered via `bundle exec fastlane ios format`) for the formatter